src/services/user.service.js

```diff
@@ -49,6 +50,55 @@ export async function createUser(fields){
     return user
 }
 
-export async function updateUser(evt){
-    let connection = await getConnection()
-}
\ No newline at end of file
+/*
```

Comment:
I think we can probably clean out this older code if it is no longer used?

Reply:
@ryanolee ok

Reply:
@ryanolee Oh actually, in here, I’m thinking about gradually moving the currently exported methods (e.g. getOneOrCreateByGoogleDetails, getUserRepository, createUser) into the userService factory function and make use of dependency injection to inject dependencies into the userService factory function (instead of merely importing with import statements at the beginning of the user.service.js file).  I’m thinking that this way, it might be easier to run unit tests later on without heavily using mocks/stubs.  Do you think this approach is alright? Or should I revert back to the more familiar approach of depending on import statements?